Applying_same_text_modification_in_several_lines/ans.q

```diff
@@ -1 +1 @@
-`:out 0:{ssr[36_x;"))";")"]}@'read0`:inp
+`:out 0:{x _ x?")"}@' 36_'read0`:inp
```

Comment:
Nice solution! But could you remove the white space after `_` and `@'` since it is a golf challenge 🙂. Also you shouldn’t need to edit the README file as it’s generated automatically!